src/store/app/app.slice.ts

```diff
@@ -10,6 +11,19 @@ import { AppState } from './app.state';
 
 const initialState = { ...new AppState() };
 
+const isLocalAccountCreated = (
```

Comment:
I'm not sure about that variable name. Maybe something like `hasLocalTransactions` would be better?